extensions/oplogPopulator/OplogPopulatorTask.js

```diff
@@ -10,10 +10,12 @@ werelogs.configure({ level: config.log.logLevel,
 
 const mongoConfig = config.queuePopulator.mongo;
 const oplogPopulatorConfig = config.extensions.oplogPopulator;
+const activeExtensions = Object.keys(config.extensions);
```

Comment:
not correct : should only include the extensions which are actually using oplogPopulator...

e.g. nothing for now :-/ ....though we may enable it for bucket notification already anyway, since we are transitioning it and it is not yet used in 8.x. This will make testing easier, with no product impact in the meantime.

Reply:
In addition, maybe we may need to check a bit deeper?
I am thinking of the case (which may not be possible, to check) where the config contains some details about an extension, but it is not fully enabled (e.g. something like `config = { "extensions": [ "replication": { "enabled": false } ] }`)